doc/source/index.rst

```diff
@@ -1,20 +1,59 @@
-.. nomenclature documentation master file, created by
-   sphinx-quickstart on Thu Nov 18 09:33:32 2021.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+**nomenclature**: Working with IAMC-style scenario data
+=======================================================
 
-Welcome to nomenclature's documentation!
-========================================
+Release v\ |version|.
```

Comment:
This line shows _Release v0.2.dev12+g880ca57._. Not sure if that is intended or if I did something wrong in the implementation of the dynamic version tracking.

Reply:
This is expected behavior: Sphinx bumps the smallest (in the sense major -> minor -> patch) part of latest version number by 1 and adds `.dev`, then the number of commits since the latest release, and then the short hash of the latest commit. If there are uncommitted changes, it also adds the term `dirty`.

Reply:
Thanks for the explanation!